docs/src/main/tut/oneand.md

```diff
@@ -21,10 +21,4 @@ type NonEmptyList[A] = OneAnd[List, A]
 
 which is the actual implementation of non-empty lists in cats.  By
 having the higher kinded type parameter `F[_]`, `OneAnd` is also able
-to represent other "non-empty" data structures, e.g.
```

Comment:
I think that after this PR (or possibly as a part of this PR) we can probably rip out `OneAnd` as discussed [here](https://github.com/typelevel/cats/issues/537#issuecomment-222667534). Someone can follow up with a `NonEmptyStream` if it's something they really want -- I suspect it's not used as commonly as `NonEmptyList` and `NonEmptyVector`.